### PR TITLE
[ICDS Dashbaord] Spelling corrections in infantomter and stadiometer

### PR DESCRIPTION
**Pull request by challabeehyv**
https://dimagi-dev.atlassian.net/browse/QA-898

##### SUMMARY
Spelling corrections in infantomter and stadiometer

##### FEATURE FLAG
No

### CODE REVIEW

**Review by Rohit25negi — APPROVED** [review]
Looks good. Hey was there a particular reason to close the last PR for this change?

**Comment by challabeehyv** [comment]
> Looks good. Hey was there a particular reason to close the last PR for this change?

i am facing issues while merging with master on submodules and not fixing with rebasing

File: custom/icds_reports/static/js/directives/kpi/kpi.directive.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ function KpiController($rootScope, $location, haveAccessToFeatures) {
 
     this.isNumber = window.angular.isNumber;
 
+    // Added to hide Infantometer and Stadiometer cards in the UI. To be removed post testing
+    this.toShowInKpi = function (cellLabel) {
+        return !cellLabel.includes('Infantometer') && !cellLabel.includes('Stadiometer');
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: hey why its not including all checks as was applied before `(cell.label !== 'AWCs Reported Infantometer' && cell.label !== 'AWCs Reported Stadiometer' && cell.label !== 'Infantometer' && cell.label !== 'Stadiometer'`

Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: @challabeehyv

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left one questions. Should be good to merge if it comes out to be not required.

File: custom/icds_reports/static/js/directives/kpi/kpi.directive.js (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: @Rohit25negi We are hiding cards with string match infantomter and stadiomter

Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: I meant why this check is not needed anymore `cell.label !== 'AWCs Reported Infantometer'`

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
can you check I have a followup question

File: custom/icds_reports/static/js/directives/kpi/kpi.directive.js (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: @Rohit25negi we are checking for any text that have Infantomter and Stadiometer which should cover this case.

**Comment by challabeehyv** [comment]
> can you check I have a followup question

@Rohit25negi we are checking for any text that have Infantomter and Stadiometer which should cover this case.

File: custom/icds_reports/static/js/directives/kpi/kpi.directive.js (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: Aaah OK. Sounds good. Thanks

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**